app/docker-compose.development.yml

```diff
@@ -0,0 +1,25 @@
+# docker-compose.yml
+
+version: "3"
+
+services:
+  app:
+    image: docker-nextjs-dev
+    build:
+      dockerfile: Dockerfile.development
+    ports:
+      - "3000:3000"
+    networks:
+      - stagingnetwork
+    volumes:
+      - ./pages:/app/pages
+      - ./public:/app/public
+      - ./styles:/app/styles
+
+networks:
+  stagingnetwork:
```

Comment:
Here is the issue, if you don't run the NextJS container inside the `oesnet` and `spnet`, there is no way for this NextJS container to connect with either the Open Event Server which is running in the `oesnet`, and the Development Fauna Container running in the `spnet`

By defining the different networks we are trying to achieve network isolation, so that any suspicious user won't be able to connect to the backend services from outside, such as the Open Event Server's PostgresDB.

Reply:
Thank you so much for pointing out the exact problem . Was stuck on this one for pretty long time